R/xy-datamapping.R

```diff
@@ -29,13 +78,129 @@ XYEDataMapping <- R6::R6Class(
 
     # Example of how to do some other stuff
     initialize = function(x, y,
-                          error = NULL,
-                          errorMin = NULL,
-                          errorMax = NULL) {
-      super$initialize(x, y)
+                              error = NULL,
+                              errorMin = NULL,
+                              errorMax = NULL, ...) {
+      super$initialize(x, y, ...)
       self$error <- error
       self$errorMin <- error %||% errorMin
       self$errorMax <- error %||% errorMax
+    },
+
+    getMapData = function(data, metaData) {
+      x <- data[, self$x]
+      y <- data[, self$y]
+      errorMin <- data[, self$errorMin]
+      errorMax <- data[, self$errorMax]
+
+      color <- getDefaultCaptions(

```

Comment:
Is there a reason why the code is indented like so? It's ok to use more than 20 columns width for code :)